frontend/src/pages/Register.css

```
@@ -0,0 +1,103 @@
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  padding: var(--margin-top-bottom) var(--margin-left-right);
+  gap: 24px;
+  background: var(--fw-bg);
+}
+
+.register-picker__heading {
+  font-family: 'Geologica', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geologica" (font-family-name-quotes)

(font-family-name-quotes)